src/csvcubed/utils/sparql_handler/sparql_queries/select_codelist_primary_key_by_dataset_url.sparql

```diff
@@ -0,0 +1,18 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX csvw: <http://www.w3.org/ns/csvw#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+
+SELECT DISTINCT ?tablePrimaryKey
+WHERE {   
+
+    ?table csvw:tableSchema ?tableSchema;
+           csvw:url ?tableUrl.
+           
+    ?tableSchema csvw:primaryKey ?tablePrimaryKey.
```

Comment:
What happens if the primary key is composed of more than one column?

Reply:
As discussed, composite primary keys are now handled.